low-level/framework/utils/sysfs_interface.py

```diff
@@ -41,8 +44,59 @@ def __init__(self):
         self.sas_phy_dir_list = []
         self.phy_dir_dict = {}
         self.sas_phy_dir_path = {}
+        self.host_base_dir = None
+        self.host_type = OSUtils.get_host_type()
+
+
+    def get_host_list(self):
+        """
+        Returns host list from host base directory.
+        /sys/class/fc_host or /sys/class/scsi_host
+
+        Example: ['host1', 'host2']
+        """
+
+        hosts = []
+
+        # Refresh host_type
+        self.host_type = OSUtils.get_host_type()
+
+        if self.host_type:
+            self.host_base_dir = os.path.join(self.sysfs, self.host_type)
+            #self.host_base_dir = os.path.join('/tmp/class', self.host_type)
```

Comment:
remove unused/commented code.

Reply:
I used that for simulation. Sure, I will remove it.

Reply:
Done.